cmd/skywire-updater/commands/gen-config.go

```diff
@@ -6,11 +6,10 @@ import (
 	"os"
 	"path/filepath"
 
-	"github.com/spf13/cobra"
-	"gopkg.in/yaml.v2"
-
 	"github.com/skycoin/skywire-updater/internal/pathutil"
```

Comment:
We should probably make this module public in `skycoin/skywire`. (Our bad!)

Reply:
Moved pathutil from /internal/pathutil to /pkg/util/pathutil as per https://github.com/skycoin/skywire/pull/295